Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG conf_dir=configuration
+ARG hub_app
```

Comment:
I don't think we want `ARG` lines before `FROM`.  They are only visible in the `FROM` line itself, and the `FROM` doesn't reference them.

https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
```suggestion
```